### PR TITLE
Exclude SLE12 platform.

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

**Comment by aplanas** [comment]
Depends on crowbar/barclamp-crowbar#1175

**Comment by tboerger** [comment]
LGTM

File: crowbar_framework/app/models/deployer_service.rb [hunk 1]
```diff
@@ -28,7 +28,10 @@ def role_constraints
         "deployer-client" => {
           "unique" => false,
           "count" => -1,
-          "admin" => true
+          "admin" => true,
+          "exclude_platform" => {
+            "suse" => "12.0"
```
Comment (vuntz) [inline crowbar_framework/app/models/deployer_service.rb]: Why? This is a basic role that all nodes need.

**Comment by aplanas** [comment]
I was wrong understanding this. I will close all the commits that reference crowbar-related barclamps.

**Pull request closed without merging**